index.html

```diff
@@ -0,0 +1,184 @@
+            </div>
+
+        </section>
+        <section class="main-program">
+            <h2>Main program</h2>
+            <div class="main-program-container">
+                <div class="box">
+                    <img width="70px" height="100px"
+                        src="images/kissclipart-lecturer-icon-clipart-lecturer-computer-icons-55c69a33907f2b16.png"
+                        alt="">
+                    <h4 class="box-main">Lectures</h4>
+                    <div class="box-text">
+                        Get adressed by some of the best musician from all around the world
+                    </div>
+                </div>
+                <div class="box">
+                    <img width="70px" height="100px" src="images/exhibition.png" alt="">
+                    <h4 class="box-main">Exhibition</h4>
+                    <div class="box-text">
+                        Appreciate various creations appluing Cc license of artists organized from art center
+                    </div>
+                </div>
+                <div class="box">
+                    <img width="70px" height="100px" src="images/forum-logo-icon-1.jpg" alt="">
+                    <h4 class="box-main">Forum</h4>
+                    <div class="box-text">
+                        Have a time to share your opinions and thoughts with expert on certain things.
+                    </div>
+                </div>
+                <div class="box">
+                    <img width="70px" height="100px" src=" images\fun-logo-vector-smile-emotion-260nw-344718320.webp"
+                        alt="">
+                    <h4 class="box-main">Fun</h4>
+                    <div class="box-text">
+                        It sure'd be fun with lots of musicians come together and perform
+                    </div>
+                </div>
+                <div class="box">
+                    <img width="70px" height="100px"
+                        src="images\money-logo-design-vector-illustrative-260nw-1810454230.webp" alt="">
+                    <h4 class="box-main">Free</h4>
+                    <div class="box-text">
+                        Pay no price whatsoever your punctuality is all that matters about getting in.
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="featured-speaker-container">
+            <h1 class="featuring">Featuring</h1>
+
+            <div class="feature-box-container">
+                <div class="feature-box">
+                    <img class="image cimage" src="" alt="">
+                    <div class="featur-box-content">
+                        <div class="feature-box-text cname"></div>
+                        <div class="feature-box-text1 cdescription"></div>
+                        <div class="feature-box-text2 coverview">
+                        </div>
+                    </div>
```

Comment:
- [ ] The features are overflowing on the tablet screen please try to fix that your  website should be responsive on all the screen devices 
![image](https://user-images.githubusercontent.com/61790587/175139270-06a4c1a3-7b84-451f-810f-415476362d91.png)